tools/update-tester/src/testrunner.rs

```diff
@@ -69,6 +98,43 @@ fn connect_to(config: &ConnectionConfig<'_>) -> TestClient {
     TestClient(client)
 }
 
+pub fn update_to_and_validate_new_toolkit_version(root_config: &ConnectionConfig) {
+    let expected_results: Vec<Vec<Option<String>>> = vec![vec![
```

Comment:
Do you mind moving this to right under the `// run validation tests` line? It took me a couple seconds to figure out where we use this data